helix-core/src/test/java/org/apache/helix/integration/multizk/TestMultiZkHelixJavaApis.java

```diff
@@ -222,6 +232,17 @@ public void afterClass() throws Exception {
       } else {
         System.clearProperty(MetadataStoreRoutingConstants.MSDS_SERVER_ENDPOINT_KEY);
       }
+
+      boolean status = false;
```

Comment:
Please rebase, I think this change has been in.

Reply:
rebased.